ibtk/include/ibtk/LibMeshSystemVectors.h

```diff
@@ -158,14 +134,17 @@ class LibMeshSystemVectors
     void zero(const std::string& vec_name);
 
 protected:
-    /// Check if a vector exists and, if not, add it.
-    void maybeAdd(const std::string& vec_name);
+    LibMeshSystemVectors(const std::vector<bool>& part_mask, std::string system_name);
 
-    /// Add an IB ghosted vector.
-    std::vector<std::unique_ptr<libMesh::PetscVector<double> > >& maybeAddIBGhosted(const std::string& vec_name);
+    /// Returns false if a standard libMesh vector; otherwise returns true.
+    inline bool vec_stored_in_map(const std::string& vec_name)
```

Comment:
We should make this static so that its clear that it doesn't depend on the object.